pennylane_qiskit/converter.py

```diff
@@ -1225,19 +1225,19 @@ def _expr_eval_clvals(clbits, clvals, expr_func, bitwise=False):
     return condition_res
 
 
-def load_noise_model(noise_model, **kwargs) -> qml.NoiseModel:
+def load_noise_model(
+    noise_model, verbose: bool = False, decimal_places: Union[int, None] = None
+) -> qml.NoiseModel:
```

Comment:
Afaik, we have been using type hints when possible in the functions in the `converter.py` since it's massive and it is the least we can do for maintainability.